pyannote/audio/core/task.py

```diff
@@ -281,29 +682,26 @@ def specifications(
     ):
         self._specifications = specifications
 
+    @property
+    def has_prepared_data(self):
+        # This flag indicates if data for this task was generated, and
+        # optionally saved on the disk
+        return getattr(self, "_has_prepared_data", False)
+
+    @has_prepared_data.setter
+    def has_prepared_data(self, value: bool):
+        self._has_setup_metadata = value
+
```

Comment:
@clement-pages are you 100% sure this is what you wanted to do?
Setting `has_prepared_data` seems to update `_has_setup_metadata`

Reply:
I think this `has_prepared_data` / `has_setup_metadata` thing is confusing....